crashlogs.cpp

```diff
@@ -153,23 +172,59 @@ namespace glaiel::crashlogs {
         cv.wait(lk, [] { return status != program_status::crashed; });
     }
 
+    //Try to get the string representation of a signal identifier, return an empty string if none is found.
+    //This only covers the signals from the C++ std lib and none of the POSIX or OS specific signal names!
+    static std::string_view try_get_signal_name(int signal) {

```

Comment:
return a string to avoid needing to bring in <string_view>, signal codes are short enough they wont use the heap on any systems with short-string-optimizations. could also just return a const char* here